fakestorage/object.go

```diff
@@ -934,17 +934,31 @@ func (s *Server) patchObject(r *http.Request) jsonResponse {
 	vars := unescapeMuxVars(mux.Vars(r))
 	bucketName := vars["bucketName"]
 	objectName := vars["objectName"]
-	var metadata struct {
+
+	type acls struct {
+		Entity string
+		Role   string
+	}
+
+	var dataInBody struct {
```

Comment:
nit: can we call this `payload`?
```suggestion
	var payload struct {
```